rocAL/rocAL_pybind/examples/rocAL_example_doc/Video_decoder.ipynb

```diff
@@ -0,0 +1,286 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d7ff9965",
+   "metadata": {},
+   "source": [
+    " ## Video Pipeline Reading From Multiple Files in rocAL\n",
+    "\n",
+    "This example presents a simple rocAL video pipeline that loads and decodes  video data. Illustrated below how to create a pipeline, set_outputs, build, run the pipeline and enumerate over the results."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b8b1a3e9",
+   "metadata": {},
+   "source": [
+    " ## Common Code"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8ac44489",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import torch\n",
+    "from amd.rocal.pipeline import Pipeline\n",
+    "import amd.rocal.fn as fn\n",
+    "import amd.rocal.types as types\n",
+    "import numpy as np\n",
+    "import os\n",
+    "%matplotlib inline\n",
```

Comment:
@SundarRajan28 Is this line required or generated?

Reply:
%matplotlib inline is required to view the output images in the jupyter notebooks. Its a special keyword used in jupyter notebooks to display the plot images